include/igl/ray_mesh_intersect.cpp

```diff
@@ -12,6 +12,41 @@ extern "C"
 #include "raytri.c"
 }
 
+namespace igl {
+template <
+  typename DerivedV,
+  typename DerivedF>
+IGL_INLINE bool ray_triangle_intersect(
```

Comment:
should document this in the header.

Reply:
Guess it makes sense to also have the vectors's scalar type as a template parameter?

Reply:
Added to the header.